src/cmd/root.go

```diff
@@ -218,11 +226,6 @@ func migrate(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
-	if cmdName, completionCmdName := cmd.Name(), completionCmd.Name(); cmdName == completionCmdName {
```

Comment:
If this is moved out of the function, then the `cmd` and `args` parameters should also be removed.

However, I had deliberately kept this condition inside the function to make the code a bit more robust against future refactorings, and keeping the clutter out of the upper level functions.

Reply:
Changed in https://github.com/containers/toolbox/pull/1252